src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/CalculateEarnings/CalculateEarningsCommandHandler.cs

```diff
@@ -26,6 +26,7 @@ public class CalculateEarningsCommandHandler : ICommandHandler<CalculateEarnings
         public async Task Handle(CalculateEarningsCommand command, CancellationToken cancellationToken = default)
         {
             var incentive = await _domainRepository.Find(command.ApprenticeshipIncentiveId);
+            if (incentive.Withdrawn) return;
```

Comment:
It's the application that is withdrawn not the incentive.  That get's deleted when the application is withdrawn.